static/js/libs/smooth-scroll.js

```diff
@@ -30,47 +30,47 @@
   };
 
   var startScroll = function startScroll() {
-    var href = this.attributes.href.nodeValue.toString(),
-        url = href.substr(0, href.indexOf('#')),
-        id = href.substr(href.indexOf('#')+1),
-        target = document.getElementById(id),
-        startY = getCurrentScroll(),
-        targetY = getTargetOffsetFromTop(target);
+    // var href = this.attributes.href.nodeValue.toString(),
```

Comment:
@uniphil can you explain why you have commented out this library? is any of this functionality now used? if not, can we just remove the whole library? tbh I haven't investigated what it used to do, and what impact this change will have on the site experience.

Reply:
Short version: We should remove the whole thing! But this change minimizes risk before doing a demo. I plan to follow up :)

--

Yeah! So, back in the DG days, they really wanted smooth scroll, so I ended up [adapting this one](https://github.com/uniphil/smooth-scroll) since there weren't a lot of good non-jquery options.

When I was working on these changes, that smooth scrolling was really broken in firefox, and while it's a little subjective, I'm pretty anti-scroll-hijacking. Especially on low-power mobile devices, it's likely to get in the way more than help.

I was going to take it out completely, but there are unfortunately a couple bits still linked into scroll behaviour (main navigation link highlighting on the home page, specifically), so I went instead for turning off the bad part with the smallest risk of breaking anything else.

(I don't love committing commented-out code, but I was rushing a bit to get this batch of changes through for the pres tomorrow)

It's high on my list of follow-ups to clean this up properly in a follow-up soon!